package.json

```diff
@@ -17,10 +17,10 @@
     "intercept-stdout": "0.0.2",
     "mocha": "^2.1.0",
     "mocha-lcov-reporter": "0.0.1",
+    "sinon": "latest",
```

Comment:
I'd rather freeze sinon and sinon-chai to a specific version - we ran into an issue last time with two versions not being compatible.